csrc/scheduler/normalization_utils.cpp

```diff
@@ -684,17 +684,24 @@ getOptionalInnerOuterPersistentBufferBatches(
   const int64_t batch_min = getMinimumBatch();
   const int64_t batch_max = getMaximumInnerOuterPersistentBufferBatch();
 
+  // increase batch_max by 1 to allow a small amount of register spills to keep
+  // a small threads_per_block so each SM can host more blocks. This can
+  // increase performance by 10% for edge cases, e.g. hidden size in the range
```

Comment:
Why are you referencing explicit dimensions of layer norm in a generic normalization scheduler? This isn't the "layer norm scheduler", do you have an intuition for all possible graphs this scheduler could be applied to?

Reply:
In fact, do you know every current benchmark that takes this path?

Reply:
This combinedInnerOuter scheduler is only used by layer norm backward and RMS norm backward. I was trying to give a solid example using layer norm since it is the most important application of this scheduler. The modifications made to this function were sourced from #566. I'll retract these changes from the current PR and will review #566 once this is finalized since #566 may be related to how we use shared memory.